code/datums/traits/negative.dm

```diff
@@ -49,13 +49,13 @@
 	name = "Blind"
 	desc = "You are completely blind, nothing can counteract this."
 	icon = "eye-slash"
-	value = -9
+	value = -6
 	gain_text = span_danger("You can't see anything.")
 	lose_text = span_notice("You miraculously gain back your vision.")
 	medical_record_text = "Patient has permanent blindness."
 
 /datum/quirk/blindness/add()
-	quirk_holder.become_blind(ROUNDSTART_TRAIT)
```

Comment:
This is gonna make them not blind anymore

Reply:
out of date